mlflow/utils/proto_json_utils.py

```diff
@@ -11,6 +11,8 @@
 
 def message_to_json(message):
     """Converts a message to JSON, using snake_case for field names."""
+    # Note protobuf encodes int64, fixed64, and uint64 to string:
+    # https://developers.google.com/protocol-buffers/docs/proto3#json
     return MessageToJson(message, preserving_proto_field_name=True)
```

Comment:
Added this comment for those encountering the same issue in the future.

Reply:
Thank you!

Reply:
Once https://github.com/mlflow/mlflow/pull/5010 is merged, we no longer need this comment.